### ISSUE TITLE
A question about node classification

### ISSUE THREAD

**Issue by jweihe** [THUDM/GraphMAE#36]
good job! But I have a question:
Some nodes are not visible when training the linear classifier because the pre-training takes these masks out, 
even when the nodes are classified, the network weights do not take these mask nodes into account, 
so why is it still good for node classification?
Can you explain this problem?

**Comment by THINK2TRY** [comment]
Hi @jweihe, thanks for your attention! There may be some misunderstandings, and I would like to clarify them.

First, the masking operation is only conducted during the self-supervised learning stage (or called pre-training). In each iteration, we randomly select a portion of nodes and mask their features. 

Then after the SSL, we obtain a pre-trained GNN.  The graph without any masking is fed into the pretrained GNN to generate embeddings for all nodes.  And the embeddings are used to train a linear classifier.

Hope this help !
